lib/screens/v2/candidate_generator/pre_fare.ex

```diff
@@ -62,15 +65,26 @@ defmodule Screens.V2.CandidateGenerator.PreFare do
         now \\ DateTime.utc_now(),
         elevator_status_instances_fn \\ &Widgets.ElevatorClosures.elevator_status_instances/2
       ) do
-    [fn -> elevator_status_instances_fn.(config, now) end, fn -> placeholder_instances() end]
+    [
+      fn -> header_instances(config, now) end,
+      fn -> elevator_status_instances_fn.(config, now) end,
+      fn -> placeholder_instances() end
+    ]
     |> Task.async_stream(& &1.(), ordered: false, timeout: :infinity)
     |> Enum.flat_map(fn {:ok, instances} -> instances end)
   end
 
+  defp header_instances(config, now) do
+    %Screen{app_params: %PreFare{header: %CurrentStopId{stop_id: stop_id}}} = config
```

Comment:
Are we assuming the `stop_id` in the header config will be populated with the full name of the station? If so, it may make more sense to use `CurrentStopName.t()` instead. Either that or use `CurrentStopId.t()` and make a V3 call to get the name of the station.

Reply:
I wonder if we could combine them by fetching from V3 if the given value is an integer or contains "place-". That should cover station IDs. If it isn't one of those, we just use the string.